client/src/components/ViewModeToggle/ViewModeToggle.scss

```diff
@@ -0,0 +1,122 @@
+.viewmode-dropdown-toggle {
+  margin-left: auto;
+  font-size: 13px;
+
+  #splitModeButton {
+    &:disabled {
+      pointer-events: all;
+    }
+  }
+
+  .viewModeToggle-chosenView-title {
+    font-size: 10px;
+    line-height: 10px;
```

Comment:
If there's an appropriate line-height variable in _variables.scss or _bootstrap-variables.scss in admin that'd be ideal to use